consensus/src/ordered_broadcast/engine.rs

```diff
@@ -459,6 +455,12 @@ impl<
                 },
             }
         }
+
+        // Close all journals, regardless of how we exit the loop
```

Comment:
The closing of channels we rely on could have caused us to exit without closing the journals. This wouldn't create a correctness issue (given how we use `sync()`) but would prevent graceful handling.